File: Java_Course/src/App/Inheritance.java
package App;

public class Inheritance {

	public static void inheritance(String [] args) {
		
		Chef normalChef = new Chef();
		normalChef.makeChicken();
		normalChef.makeSalad();
		normalChef.makeSpecialDish();
		
		ItalianChef italianChef = new ItalianChef();
		italianChef.makeSpaghetti();
		italianChef.makeChicken();
		italianChef.makeSpecialDish();
		
		ChineseChef chineseChef = new ChineseChef();
		chineseChef.makeFriedRice();
		chineseChef.makeChicken();
		chineseChef.makeSpecialDish();
	}
}

File: C_Sharp_Course/C_Sharp_Course/Files/inheritance.cs
using C_Sharp_Course.Classes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.CompilerServices;
using System.Text;
using System.Threading.Tasks;

namespace C_Sharp_Course
{
    class UsingInheritance
    {
        static void Inheritance(string[] args)
        {
            Chef chef = new Chef();
            chef.MakeChicken();
            chef.MakeSpecialDish();

            Console.WriteLine();

            ItalianChef italianChef = new ItalianChef();
            italianChef.MakeChicken();
            italianChef.MakePasta();
            italianChef.MakeSpecialDish();

            Console.WriteLine();

            Console.ReadLine();
        }
    }
}